sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubRowToMessageTest.java

```diff
@@ -820,6 +817,71 @@ public void testSchemaReflection_matchesAll() {
             FieldMatcher.of(STRING_FIELD.getName(), TypeName.INT16)));
   }
 
+  @Test
+  public void testSchemaReflection_matchesAny() {
+    SchemaReflection schemaReflection = SchemaReflection.of(ALL_DATA_TYPES_SCHEMA);
+    assertTrue(
+        schemaReflection.matchesAny(
```

Comment:
should this be `matchesAll`? (not familiar with the utility)

Reply:
The test `.._matchesAny` evaluates `SchemaReflection.matchesAny`.